src/BugsnagLogger.php

```diff
@@ -17,6 +18,13 @@ class BugsnagLogger extends AbstractLogger
     protected $client;
 
     /**
+     * The minimum level required to notify bugsnag.
```

Comment:
There should be a note here to indicate that under the threshold, log messages are converted to breadcrumbs.